metabase/metabase
# Replace pinned collection items table with pinned overview section

Related to #19758. This is merging into a feature branch.

This PR replaces the pinned items table with an overview section. All the same functionality should exist, except for sorting -- the pinned items are now sorted alphabetically without recourse (may end up implementing drag and drop later).

If you look at the product docs/mocks you'll still see a few differences, namely:
1. other various UI changes such as the background color
2. navigating directly to an ad hoc query builder view when a user clicks on a Model card
3. in the mocks, cards show visualizations -- that's coming in a separate PR

These are all getting handled in different PRs.

ALSO, I'm removing a bunch of cypress tests that no longer make sense. Will address fully testing this stuff in a separate PR 🙂

**Testing**
1. navigate to collections (ie `/collection/root`) and pick a test collection (or make one)
2. if you have no pinned items, click on a table item on the page (assuming there are items) and click the pin action ("Pin this item")
3. the item should show up above the table now
4. repeat this for questions, dashboard, models
5. hovering over the cards makes an ellipsis menu appear; test the actions out
6. click on the cards, make sure that works, etc.

lil demo:

https://user-images.githubusercontent.com/13057258/150240238-45c26199-3647-49d4-ba2c-9f68ef8786ce.mov
